### PR TITLE
Package ocamlformat.0.11.0

### PR DESCRIPTION
**Pull request by gpetiot**
### `ocamlformat.0.11.0`
Auto-formatter for OCaml code
OCamlFormat is a tool to automatically format OCaml code in a uniform style.

---
* Homepage: https://github.com/ocaml-ppx/ocamlformat
* Source repo: git+https://github.com/ocaml-ppx/ocamlformat.git
* Bug tracker: https://github.com/ocaml-ppx/ocamlformat/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/ocamlformat/ocamlformat.0.11.0/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ["ocaml" "tools/gen_version.mlt" "src/Version.ml" version] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.06"}
+  "base" {>= "v0.11.0"}
+  "base-unix"
+  "bisect_ppx"
+  "cmdliner"
+  "dune" {build & >= "1.1.1"}
```
Comment (kit-ty-kate) [inline packages/ocamlformat/ocamlformat.0.11.0/opam]:
```suggestion
  "dune" {>= "1.1.1"}
```
Dune should not be flagged as a build dependency anymore (see #14266)

**Comment by kit-ty-kate** [comment]
The rest is fine. Once the dune thing will be fixed it will be good to merge.

**Comment by gpetiot** [comment]
> The rest is fine. Once the dune thing will be fixed it will be good to merge.

The opam file is fixed, I don't know why the last CI check is still pending.

**Comment by kit-ty-kate** [comment]
Me neither. Thanks!

**Pull request merged**